package.json

```diff
@@ -18,11 +18,12 @@
     "lint:ts": "tslint -p .",
     "lint:style": "stylelint 'app/**/*.scss' 'lib/**/*.scss'",
     "lint:template": "ember-template-lint 'app/**/*.hbs' 'lib/**/*.hbs'",
-    "test": "EMBER_ENV=test ember test",
-    "test:cover": "EMBER_ENV=test COVERAGE=true ember test",
+    "test": "EMBER_ENV=test ember exam --split=4 --parallel=1",
```

Comment:
would adding the `--random` option here be worthwhile? or does the --split already ensure that it's randomized?

Reply:
I am open to turn the flag on. `--split=x` does not randomize by default. It just places tests in `x` buckets (partitions). The only issue is that you might have to fix a weird probabilistic failure in order for your PR to be mergeable. This is not a bad thing but it may add overhead/stress when it's crunch time. I was thinking it might be best to have a separate workflow that runs periodically, say every Sunday, with the flag turn on. Then we can create a maintenance ticket to fix those tests as they arise.

Reply:
That's a good point. I'd like to address any weird test failures, but definitely when appropriate. I think your proposed solution sounds good 👍